WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListItem.kt

```diff
@@ -121,4 +123,8 @@ sealed class ActivityLogListItem(val type: ViewType) {
             fun fromValue(value: String?) = map[value] ?: DEFAULT
         }
     }
+
+    enum class SecondaryAction(val itemId: Long) {
+        RESTORE(0), DOWNLOAD_BACKUP(1)
```

Comment:
1) Minor (🔍 ): Consider having the enum values in separate lines.
2) Minor (🔍 ): Consider adding the explicit `;` character to terminate the last enum. This way if more logic is added to this enum later on there would be no line change diffs there.